three-tier-main.tf

```diff
@@ -172,7 +174,7 @@ resource "aws_security_group" "web-sg" {
     from_port   = 80
     to_port     = 80
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure Security Groups do not have unrestricted specific ports open - (HTTP,80)**
 
**Vulnerability**
Security Groups-Unrestricted Specific Ports -HTTP,80. Exposed ports may allow attackers to access the application listening on that port.
 
**Remediation**
In AWS Console - 
 1. Sign in the AWS Console and go to the Amazon VPC console.
 2. In the navigation pane, select Security Groups.
 3. For each security group, perform the following actions:
      a) Select the security group.
      b) Select the Inbound Rules.
      c) Identify the rules to be removed.
      d) Remove the required rules.
 4. Click Save.

source: tenablecs-